asf-site-src/source/documentation/in-progress/precommit/plugins/golangcilint.html.md

```diff
@@ -0,0 +1,53 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Name
+
+golangcilint
+
+# Category
+
+Test
+
+# Description
+
+NOTE: Go support is experimental.  Additionally, this plug-in only supports Go using Go modules.
```

Comment:
nit: I would love it if every use of "foo is experimental" included a call to action with a link to our "how to contribute" guide

Reply:
We should probably do something here, but I'm not sure what.  i.e., what does experimental actually mean? etc.